tests/test_getter_setter.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from six import text_type, u, b, PY2, PY3
+from nose.tools import raises
+from expects import *
+
+from opentok import OpenTok, __version__
+
+
+class GetterSetterTest(unittest.TestCase):
+    def setUp(self):
+        self.api_key = u("123456")
+        self.api_secret = u("1234567890abcdef1234567890abcdef1234567890")
+        self.session_id = u(
+            "1_MX4xMjM0NTZ-flNhdCBNYXIgMTUgMTQ6NDI6MjMgUERUIDIwMTR-MC40OTAxMzAyNX4"
+        )
+        self.opentok = OpenTok(self.api_key, self.api_secret)
+
+    def test_getset_appname(self):
+        assert self.opentok.app_name == "OpenTok-Python-SDK"
+        self.opentok.app_name = "Vonage-Video-SDK"
```

Comment:
As mentioned in the constructor comment, we shouldn't set a default app name.

Reply:
@dragonmantank 
- Removed default value assignment from the constructor
- Removed getters and setters for `app_name`
- Removed getters and setters test for `app_name`
- Executed tests: `nosetests -v tests/test_*` with successful output